FCCee/ALLEGRO/compact/ALLEGRO_o1_v03/ALLEGRO_o1_v03.xml

```diff
@@ -72,6 +74,8 @@
            outer_radius="20*cm">
     </field>
 
+    <!-- FFQ fields at Z pole, V23) -->
```

Comment:
Can you make the comment more clear? Why is this commented now?

Reply:
I added the magnetic fields from the first final focus quadrupoles at the Z working point. I then realized that the gradients of these magnets will vary for the 4 working points and due to the evolution of the lattice, so a smarter modular solution should be found. I commented that to leave a trace of it somewhere as a starting point, but if you think it can be removed.

Reply:
Looking at Oide's latest lattice, the normalized gradients change of about a factor 2, but to get the magnetic field one should also multiply everything for the energy of the beam (about x4 from Z to T).

```
<coefficient coefficient="(-1)*( BEAM_ENERGY )*( K1 )/0.3*tesla/m"/>
```
```
!Conversion of FCCee_t_605_nosol_13.sad at Tue May 28 10:06:16 CEST 2024
K1QC1R1 = -.1610150586107995;
LQC1R1 = .7;
K1QC1R2 = -.13901180599091467;
LQC1R2 = 1.25;
K1QC1R3 = -.14229074672195402;
LQC1R3 = 1.25;
```
```
!Conversion of FCCee_z_605_nosol_2.sad at Wed May 29 09:17:56 CEST 2024
K1QC1R1 = -.2886116378554657;
LQC1R1 = .7;
K1QC1R2 = -.20372454978144672;
LQC1R2 = 1.25;
K1QC1R3 = -.08946281314982539;
LQC1R3 = 1.25;
```

Reply:
Fine for me to keep this, and after Andrea's explanation I also understand the comment.